src/lib/dai.c

```diff
@@ -134,6 +135,50 @@ static inline const struct dai_type_info *dai_find_type(uint32_t type)
 	return NULL;
 }
 
+#ifdef __ZEPHYR__
+struct dai *dai_get_zephyr(const struct dai_type_info *dti, uint32_t type, uint32_t index,
+			   uint32_t flags)
```

Comment:
why are `flags` needed here?

Reply:
yeah, was just copy paste if needed with other drivers, but can remove and add later if needed